protocol/scripts/genesis/prod_pregenesis.sh

```diff
@@ -116,6 +116,7 @@ function overwrite_genesis_production() {
 	# Rewards params
 	dasel put -t string -f "$GENESIS" '.app_state.rewards.params.denom' -v "$NATIVE_TOKEN"
 	dasel put -t int -f "$GENESIS" '.app_state.rewards.params.fee_multiplier_ppm' -v '0'
+	dasel put -t int -f "$GENESIS" '.app_state.rewards.params.market_id' -v "$REWARDS_TOKEN_MARKET_ID"
```

Comment:
Missed this previously; `REWARDS_TOKEN_MARKET_ID` was not being used

To prevent similar mistake, using different values for these envvars to ensure they are effectivec